ml-agents/mlagents/trainers/sac/trainer.py

```diff
@@ -99,6 +100,19 @@ def __init__(
             else False
         )
 
+    def _check_param_keys(self):
```

Comment:
This code is duplicated (and not put in a higher-level class) in case we decided to add algorithm-specific behavior, or remove one of these parameters from one but not both of the algorithms.

Reply:
That makes sense. In this case, do you have an algorithm in mind where you wouldn't want to check these hyperparameters? Else, it would make sense to me to put this in a higher level class.

Reply:
Anything that doesn't support LSTM :D

Reply:
Alright.